packages/cli/src/commands/__tests__/dev.test.js

```diff
@@ -1,34 +1,37 @@
 import '../../lib/mockTelemetry'
 
-jest.mock('concurrently', () => ({
+vi.mock('concurrently', () => ({
   __esModule: true, // this property makes it work
-  default: jest.fn().mockReturnValue({
+  default: vi.fn().mockReturnValue({
     result: {
       catch: () => {},
     },
   }),
 }))
 
 // dev checks for existence of api/src and web/src folders
-jest.mock('fs', () => {
+vi.mock('fs-extra', async () => {
+  const actualFs = await vi.importActual('fs-extra')
   return {
-    ...jest.requireActual('fs'),
-    readFileSync: () => 'File content',
-    existsSync: () => true,
+    default: {
+      ...actualFs,
+      readFileSync: () => 'File content',
+      existsSync: () => true,
+    },
   }
 })
 
-jest.mock('@redwoodjs/internal/dist/dev', () => {
+vi.mock('@redwoodjs/internal/dist/dev', () => {
   return {
-    shutdownPort: jest.fn(),
+    shutdownPort: vi.fn(),
   }
 })
 
-jest.mock('@redwoodjs/project-config', () => {
-  const actualProjectConfig = jest.requireActual('@redwoodjs/project-config')
+vi.mock('@redwoodjs/project-config', async () => {
```

Comment:
Why not `, async (importActual) => {` here like in other test cases?

Reply:
I was simply copying the style of the previous code. I would like to have this sort of refactor be a follow up?

Reply:
Sounds good 👍 
Asking to 1. make sure I understand what's going on and to learn. 2. To make sure it's a deliberate choice on your part